app/controllers/course/statistics/answers_controller.rb

```diff
@@ -22,7 +22,7 @@ def latest_attempt
                                     discussion_topic: { posts: :codaveri_feedback }).first
   end
 
-  def question_answer_details
+  def attempts
```

Comment:
similarly why is this `attempts` not `answers`?

Reply:
Attempt aligns with the vocab used in frontend, since we have never exposed the concept of "answer" to users

Reply:
Sure, do leave a comment that an "attempt" is an answer that has been submitted before.

Reply:
Done